vue2-vuetify/src/dynaform/DynaformStringControlRenderer.vue

```diff
@@ -52,11 +52,7 @@
           :persistent-hint="persistentHint()"
           :required="control.required"
           :error-messages="control.errors"
-          :value="
-            control.data
-              ? control.data
-              : control.uischema.options.defaultValue || ''
-          "
+          v-model="textValue"
```

Comment:
why are you using v-model instead of :value?
Another field controls are using :value